mobile/components/SolidDropDown.tsx

```diff
@@ -24,7 +24,7 @@ export default function SolidDropDown({
 }: SolidDropDownProps) {
   const [fieldValue, setFieldValue] = React.useState(placeholder);
   const [selectedOptions, setSelectedOptions] = React.useState<Set<string>>(
-    new Set()
+    new Set("Veteran")
```

Comment:
Is there a reason why a set with the characters of veteran is the default value?